coalib/coala_modes.py

```diff
@@ -26,7 +29,8 @@ def mode_normal(console_printer, log_printer, args, debug=False):
         console_printer=console_printer,
         args=args,
         debug=debug)
-
+    if log_printer:
```

Comment:
You are using `if`  without `else` it would be good to do something like
if log_printer is None:
     pass
else:
     print the warning

Reply:
No that's redundant, don't use `pass` just to have a complete `if-else` construct. It's totally valid to have an `if` without an `else`.